<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SAMEER" failures="3" tests="5" name="edgetest" time="22.879" errors="0" timestamp="2024-09-01T17:27:59 IST">
  <testcase classname="testCases.EC003_LoginDDT" name="verify_loginDDT" time="3.411"/>
  <testcase classname="testCases.EC003_LoginDDT" name="verify_loginDDT" time="0.973">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.EC003_LoginDDT.verify_loginDDT(EC003_LoginDDT.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.EC003_LoginDDT" name="verify_loginDDT" time="0.789">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.EC003_LoginDDT.verify_loginDDT(EC003_LoginDDT.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.EC003_LoginDDT" name="verify_loginDDT" time="0.968">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.EC003_LoginDDT.verify_loginDDT(EC003_LoginDDT.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.EC003_LoginDDT" name="verify_loginDDT" time="3.766"/>
</testsuite> <!-- edgetest -->
